cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -72,7 +73,7 @@ export KUBE_CUSTOM_NETD_YAML=$(curl -s \
 
 Now bring up a cluster using one of the following two methods:
 
-#### 2.a Create a regular Kubernetes cluster
+#### 2a. Create a regular Kubernetes cluster
```

Comment:
Do we need to keep the netd instruction? It doesn't apply to windows nodes, and probably doesn't need to be in this README

Reply:
Removed.